packages/wallet/test/PersonalWallet/methods.test.ts

```diff
@@ -268,6 +271,20 @@ describe('PersonalWallet methods', () => {
         await expect(wallet.submitTx(tx, { mightBeAlreadySubmitted: true })).resolves.not.toThrow();
         expect(await txPending).toEqual(outgoingTx);
       });
+
+      it.only('resolves on success when submitting a tx when sending coins to a handle', async () => {
+        const txBuilder = wallet.createTxBuilder();
+        const txOutput = await txBuilder.buildOutput().handle('alice').coin(1_000_000n).build();
+
+        const txOut = await txBuilder.addOutput(txOutput).build().sign();
+
+        const txPending = firstValueFrom(wallet.transactions.outgoing.pending$);
+        await wallet.submitTx(txOut);
+
+        const txPendingResult = await txPending;
+
+        expect(txPendingResult.body.outputs[0].address).toEqual(mocks.resolvedHandle.resolvedAddresses.cardano);
```

Comment:
```suggestion
        expect(txPendingResult.body.outputs[0].address).toEqual(mocks.resolvedHandle.resolvedAddresses.cardano);
        // expect txSubmitProvider.submitTx to be called with handle in context
```

Reply:
2d88f9ab41c0bf941b8126409e8d597fa2713303  should address this